organizations/backends.py

```diff
@@ -34,11 +50,37 @@ def authenticate(self, *args, **kwargs):
         # superuser can log into any microsite
         site = get_current_site()
         is_default_site = site.id == settings.SITE_ID
+
         if not is_default_site and is_site_configuration_enabled() and user and not user.is_superuser:
             user_organizations = set(user.organizations.values_list('short_name', flat=True))
-            site_organization = get_value('course_org_filter')
```

Comment:
@OmarIthawi @bryanlandia Any idea why we're doing this instead of simply checking

```
if UserOrganizationMapping.objects.filter(user_id=user.id, organization__sites__in=[site]).exists():
    do_our_stuff()
```

Reply:
I believe we have followed this pattern of using `course_org_filter` because it is more performant but I will give it another think.